src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  --bigIconSize: 19px;
+  --smallIconSize: 11px;
+}
+
+.search {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.search--big {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search--small {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.search--big .search__icon {
+  display: flex;
+  background-image: url(images/Search.svg);
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat no-repeat;
+  width: 19px;
+  height: 19px;
+  min-width: var(--bigIconSize);
+  min-height: var(--bigIconSize);
+  position: relative;
+  top: 25px;
+  left: 26px;
+  margin-right: -19px;
+}
+
+.search--small .search__icon {
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 11px;
+  height: 11px;
+  min-width: 11px;
+  min-height: 19px;
+  position: relative;
+  top: 15px;
+  left: 13px;
+  margin-right: -11px;
+}
+
+.search input {
```

Comment:
Consider adding a class for the input element, use BEM methodology for class naming